src/components/graph-view.js

```diff
@@ -1581,11 +1584,18 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       this.entities.appendChild(nodeContainer);
     }
 
-    // ReactDOM.render replaces the insides of an element This renders the element
-    // into the nodeContainer
     const anyElement: any = element;
 
-    ReactDOM.render(anyElement, nodeContainer);
```

Comment:
@jasonli468 @chenyunw-uber why do we need to render the component?
Can't we just export the component and the host application take care of mounting it a DOM node?

Reply:
I don't actually take a full look at this but simply deprecate ReactDOM.render with createRoot. Otherwise we will have to change a lot.

Reply:
I agree with Chenyun, this is something we could potentially investigate but I think that we should add it to our Eng Fundamentals backlog instead of as part of this commit as it would greatly increase the scope of these changes and would also likely be a breaking change for all consumers of this package.